app/routes/events/EventCreateRoute.ts

```diff
@@ -114,6 +116,7 @@ const mapStateToProps = (state, props) => {
       }),
       registrationDeadlineHours: 2,
       unregistrationDeadlineHours: 2,
+      responsibleUsers: [],
```

Comment:
To avoid making the process of creating new events more tiresome, we could set the default value of responsibleUsers to be the author here. In that way, if the responsible user is creating the event they wouldn't have to change anything, but they have the option to do so if they desire.

Reply:
This was how it was before the latest commit so I can revert it back to that 👍